app/services/report_importer.rb

```diff
@@ -150,6 +150,11 @@ def create_report_and_logs
     # Run report scanner
     scan
 
+    # Report metric counts via telemetry
+    ConfigReportStatusCalculator.new(:bit_field => status).status.each do |metric, count|
```

Comment:
Would this work for report importers that aren't ConfigReportImporters but inherit from this class, such as omaha? https://github.com/theforeman/foreman_omaha/blob/0a5abeb5a54d95cedb92170887a65327d00eebf2/app/services/foreman_omaha/omaha_report_importer.rb#L20 
cc: @timogoebel

Reply:
I think this won't work. Good catch. Can we simply move this code into `ConfigReportImporter`?

Reply:
Good catch however that does not stop us from merging this small improvement, does it? ;)

Reply:
Let's move this to `ConfigReportImporter` or it'll break plugins.

Reply:
Ok thanks moved it here.